### DISCUSSION TITLE
Possible patch release in this week?

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1609] [Release works]
Hi @everyone,

I think it's worth releasing new patch release to solve
https://github.com/sympa-community/sympa/issues/763

**Comment by ldidry** [comment]
Sure.

**Comment by racke** [comment]
Fair enough :+1:

**Comment by ikedas** [comment]
okey, patch release will be out at 29 or 30 Sep.

We are given time to update description about RPM repo. 
  https://github.com/sympa-community/sympa/releases

**Comment by xavierba** [comment]
Done, sorry for not attending to this earlier.

**Comment by ikedas** [comment]
Thanks for update! I rather am sorry that I couldn’t help your work much.

**Comment by ikedas** [comment]
@xavierba, will epel provide EL6/i386 package of sympa? i remember someone used it.

**Comment by ikedas** [comment]
Done!

**Comment by xavierba** [comment]
> @xavierba, will epel provide EL6/i386 package of sympa? i remember someone used it.

The problem with EL6/i386 is RHEL/CentOS 6 is missing perl(Locale::Messages), which is provided by perl-libintl-perl on x86_64. That's why I was not building it in COPR and have added an exclusion for the EPEL package.
The only way to provide sympa for EL6/ix86 would be to build a limited arch rpm in EPEL (like you do in the sympa-ja.org repo). This seems like a lot work for a rather thin gain, especially given that EL6 will be EOL in 1 year from now, but this is feasible indeed.
Would you have some download stats to back the realness of the need to fill this void ?

**Comment by ikedas** [comment]
I agree to you that adding new package for epel6 now will give us little benefit.

Sympa-ja.org already have perl-libintl of el6/i386 arch.  There, I will be able to continue providing i386 sympa package for a while.

**Comment by xavierba** [comment]
Another alternative would be to use the sympa COPR. This would avoid all the hurdle of getting the missing perl module into EPEL 6. First build perl-libintl SRPM taken straigh from CentOS [1] in the COPR only for EL6/ix86, then build sympa itself.
I will try and add some sort of disable around the ExcludeArch: %{ix86} in sympa specfile, so one can still use the same SRPM to build the package on EL6/ix86 in the COPR (and possibly automate the build when the "real" sympa rpm is built in EPEL6)

[1] http://vault.centos.org/6.10/os/Source/SPackages/perl-libintl-1.20-1.el6.src.rpm

**Comment by ikedas** [comment]
@xavierba, thanks for suggestion! But I already tried removing ExcludeArch and successfully rebuilt el6/i386 sympa packages.

I’d like to go on with this method in this one year, because users may keep sympa up-to-date with least changes,

**Comment by xavierba** [comment]
I just made a change to the sympa specfile to allow forcing the build on EL6 ix86 [1].
Now one just need to add '--with el6_ix86' to either rpmbuild or mock command line with an un-modified srpm. I'm not sure what you are using to produce the sympa-ja.org rpms, but this should save you from the need to modify the spec.

I've also added the same --with option to the COPR config for the epel-6-i386 chroot, so sympa can be built straight from the Fedora's git repository. The perl-libintl srpm from CentOS 6 was indeed rebuilt previously to allow the build to succeed.

So we should be all set for either using sympa-ja.org or sympa COPR for EL6 ix86.

[1] https://src.fedoraproject.org/rpms/sympa/c/473a754024730462fc7132299c1ae7114ff45505?branch=master
